src/Razor/test/Microsoft.VisualStudio.LiveShare.Razor.Test/Host/DefaultProjectSnapshotManagerProxyTest.cs

```diff
@@ -194,6 +196,11 @@ public async Task GetStateAsync_CachesState()
             Assert.Same(state1, state2);
         }
 
+        public void Dispose()
+        {
+            JoinableTaskContext.Dispose();
```

Comment:
Doesn't the `JoinableTaskFactory` own the `JoinableTaskContext` and can't that be disposed❔

Reply:
`JoinableTaskFactory` is not Disposable.